RetailCoder.VBE/Settings/CodeInspectionConfigProvider.cs

```diff
@@ -29,61 +33,9 @@ public void Save(CodeInspectionSettings settings)
-                new CodeInspectionSetting("ObsoleteCommentSyntaxInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("ObsoleteGlobalInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("ObsoleteLetStatementInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("ObsoleteTypeHintInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("OptionBaseInspection", CodeInspectionType.MaintainabilityAndReadabilityIssues, CodeInspectionSeverity.Hint),
-                new CodeInspectionSetting("OptionExplicitInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
-                new CodeInspectionSetting("ParameterCanBeByValInspection", CodeInspectionType.MaintainabilityAndReadabilityIssues, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("ParameterNotUsedInspection", CodeInspectionType.CodeQualityIssues),
-                new CodeInspectionSetting("ProcedureCanBeWrittenAsFunctionInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("ProcedureNotUsedInspection", CodeInspectionType.CodeQualityIssues),
-                new CodeInspectionSetting("RedundantOptionInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Hint),
-                new CodeInspectionSetting("SelfAssignedDeclarationInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("UnassignedVariableUsageInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
-                new CodeInspectionSetting("UndeclaredVariableUsageInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Error),
-                new CodeInspectionSetting("UntypedFunctionUsageInspection", CodeInspectionType.LanguageOpportunities, CodeInspectionSeverity.Hint),
-                new CodeInspectionSetting("UseMeaningfulNameInspection", CodeInspectionType.MaintainabilityAndReadabilityIssues, CodeInspectionSeverity.Suggestion),
-                new CodeInspectionSetting("VariableNotAssignedInspection", CodeInspectionType.CodeQualityIssues),
-                new CodeInspectionSetting("VariableNotUsedInspection", CodeInspectionType.CodeQualityIssues),
-                new CodeInspectionSetting("VariableTypeNotDeclaredInspection", CodeInspectionType.LanguageOpportunities),
-                new CodeInspectionSetting("WriteOnlyPropertyInspection", CodeInspectionType.CodeQualityIssues, CodeInspectionSeverity.Suggestion),
-            };
+            return _foundInspections.Select(inspection => new CodeInspectionSetting(inspection));
```

Comment:
Hmm, couldn't that `IEnumerable<CodeInspectionSetting>` be readonly instead of re-iterating `_foundInspections` every time? I think we might only need a `private readonly IEnumerable<CodeInspectionSetting> _defaultSettings;` field, if we need a field at all (C#6 readonly property initialization opportunity here).